bin/db-maintenance/ensure-indexes.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/**
+ * Ensure DB indices are generated. Note that this is a heavy action for the DB.
+ *
+ * Usage:
+ *
+ *  npm run ensure-indexes
+ *
+ * Ensure indexes only for specific Mongoose Model
+ *
+ *  npm run ensure-indexes -- ModelName
+ *
+ * Example:
+ *  npm run ensure-indexes -- User
+ */
+
+const config = require('../../config/config');
+const mongooseService = require('../../config/lib/mongoose');
+
+let predefinedModel;
+if (process.argv[2]) {
+  console.log(`Ensuring indexes only for Mongo collection "${process.argv[2]}"`);
+  predefinedModel = process.argv[2];
+} else {
+  console.log('Ensuring indexes for all Mongo collections');
+};
+
+mongooseService.connect(async (connection) => {
```

Comment:
Hmm, @mrkvon do you know why eslint reports failure for this:

> [lint] /home/travis/build/Trustroots/trustroots/bin/db-maintenance/ensure-indexes.js
> [lint]   29:44  error  Parsing error: Unexpected token =>

I've added the file to migrated -list. Wondering if the separate override for `bin/*` messes something up?

Reply:
I found a `/bin/.eslintrc.js` which sets `ecmaVersion: 6`.
After deleting, a few other errors appear, in unrelated files.

In the process I learned `eslint --print-config path/to/file`. It's very useful because it prints the configuration used for the file. (to be used with `npx`).

Reply:
Good catch!

Reply:
Removed in https://github.com/Trustroots/trustroots/pull/1042